gsy_framework/sim_results/cumulative_grid_trades.py

```diff
@@ -192,11 +192,11 @@ def _update_area_children_in_accumulated_trades_dict(self, accumulated_trades, a
         Returns: None
         """
         current_child_uuids = [
-            c["uuid"] for c in accumulated_trades[area["uuid"]]["children"]
+            c["uuid"] for c in accumulated_trades[area["uuid"]].get("children", [])
```

Comment:
Nit:
```suggestion
            child["uuid"] for child in accumulated_trades[area["uuid"]].get("children", [])
```
I know this wasn't part of the PR, but it might be good to fix these situations while we refactor existing code. Anyway, it's just minor.